src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -883,12 +884,12 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
       | SERIAL_FIELD_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) FIELD_NAME)? (WS | NEWLINE)*
           ((WS | NEWLINE) FIELD_TYPE)? (WS | NEWLINE)* ((WS | NEWLINE) description)?
 
-      | SINCE_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
+      | SINCE_LITERAL (WS | NEWLINE)+ description
 
-      | THROWS_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) CLASS_NAME)? (WS | NEWLINE)*
-          ((WS | NEWLINE) description)?
+      | THROWS_LITERAL (WS | NEWLINE | {!isNextJavadocTag()}? LEADING_ASTERISK)+ CLASS_NAME
+          (WS | NEWLINE)* ((WS | NEWLINE) description)?
 
-      | VERSION_LITERAL (WS | NEWLINE)* ((WS | NEWLINE) description)?
+      | VERSION_LITERAL (WS | NEWLINE)+ description
 
       | CUSTOM_NAME (WS | NEWLINE)* ((WS | NEWLINE) description)?
```

Comment:
`description` should not be forced for custom tags I feel.

Reply:
Ok